### PR TITLE
Zipkin: import/export traceId

### PR DESCRIPTION
**Pull request by glassfox**
Motivation:

Option to import and export Zipkin context

### CODE REVIEW

**Review by tsegismont — CHANGES_REQUESTED** [review]
Thank you @glassfox 

Can you move these methods to a separate class (similar to `OpenTracingUtil`) ?
Also, can you add tests?

Thank you

**Comment by glassfox** [comment]
Sure, done.

**Comment by vietj** [comment]
can you add documentation @glassfox ?

**Comment by glassfox** [comment]
done

File: vertx-zipkin/src/main/java/io/vertx/tracing/zipkin/ZipkinTracerUtil.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    if (ctx != null) {
+      return ctx.getLocal(ACTIVE_SPAN);
+    }
+    return null;
+  }
+
+  /**
+   * @return the current active {@link TraceContext} otherwise {@code null}
+   */
+  public static TraceContext getContext() {
```
Comment (vietj) [inline vertx-zipkin/src/main/java/io/vertx/tracing/zipkin/ZipkinTracerUtil.java:27]: can you rename to currentTraceContext() ?

Comment (glassfox) [reply vertx-zipkin/src/main/java/io/vertx/tracing/zipkin/ZipkinTracerUtil.java]: done

**Comment by tsegismont** [comment]
@glassfox @vietj I had not noticed there were already some static helper methods in the original class, sorry.

@vietj I believe this is an acceptable breaking change, do you agree?

**Comment by vietj** [comment]
we should not break for a rename if we can, my bad

**Comment by tsegismont** [comment]
@glassfox @vietj I've reworked the changes to avoid the breaking changes of the API

**Pull request merged**